docs/GraFx-Studio/concepts/output-tasks/index.md

```diff
@@ -0,0 +1,95 @@
+# Output Tasks
+
+## Overview
+
+**Output Tasks** represent each job initiated to create output within GraFx Studio. This concept applies to both output jobs started from Studio UI, Template Designer Workspace and those triggered through the API.
+
+```sh
+/api/v1/environment/{environment}/output/pdf
+```
+
+Each output job is listed as an **Output Task** in the system, providing visibility into the status and details of all output jobs. The **Output Task List** allows users to monitor and manage the output tasks.
+
+## Accessing the Output Task List
+
+The **Output Task List** can be accessed through the **Environment Settings**. Here, users (Subscription Admins & Environment Admins) can view all the output jobs associated with the current environment.
+
+![screenshot-full](ot1.png)
+
+![screenshot-full](ot2.png)
+
+## Output Task List Columns
+
+The Output Task List contains several key details for each job, organized in the following columns:
+
+- **Name**: Displays the name of the document associated with the output job.
+- **Status**: Shows the current status of the output task. Possible statuses include:
+  - *Started*
+  - *Pending*
+  - *Finished*
+- **Format**: Indicates the output format, such as PDF, PNG, MP4, etc.
```

Comment:
today non-pdf output is listed here if it's a single output, there is only batch output for pdf